### ISSUE TITLE
[Bug]: Ribbon bar does not integrate correctly with form when maximized

### ISSUE THREAD

**Issue by mbagattini** [Krypton-Suite/Standard-Toolkit#1572]
**Describe the bug**
When a Ribbon control is added to a KryptonForm, it does not integrate correctly with the window chrome. Symptoms include:

* When the window is maximized, the form title is displayed twice
* AllowFormIntegrate property does not change the aspect of the window's buttons and title bar 

**To Reproduce**
1. Create a new Windows Forms project with .NET version 6.0
2. Reference Krypton, e.g. version 85.24.6.176
3. Make Form1 a KryptonForm
4. Drop a Ribbon control
5. Start the application, maximize the window

**Expected behavior**
A seamless integration between the Ribbon control and the form, in terms of colors. Form that follows the palette set for it. When maximized, window title should be displayed only once.

**Screenshots**
![image](https://github.com/Krypton-Suite/Standard-Toolkit/assets/941612/7781b888-273b-4fca-9d13-da9dae2be0fd)

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Version: 22H2
 - Framework/.NET Version: .NET 6.0
 - Toolkit Version: 85.24.6.176

**Additional context**
Issue has been discussed here: #1549
The discussion also contains a sample program

**Comment by Wagnerp** [comment]
Hi @mbagattini & @giduac

One thing to note that in V90, we have completely ripped out the old Aero stuff, as it doesn't work well with Windows 10/11. That's probably why there's an inconsistency between V80/85 & V90.

**Comment by giduac** [comment]
@Wagnerp 

Now what shall we do, have @mbagattini use a v90 canary build?

**Comment by Wagnerp** [comment]
> @Wagnerp
> 
> Now what shall we do, have @mbagattini use a v90 canary build?

@giduac

If @mbagattini can test again using canary build .176 then it should solve this issue.

**Comment by giduac** [comment]
@Wagnerp 

So a fix for V85 is not realistic atm?

**Comment by Wagnerp** [comment]
> @Wagnerp
> 
> So a fix for V85 is not realistic atm?

@giduac

No, as a lot of code + fixes had to be redone.

**Comment by mbagattini** [comment]
> If @mbagattini can test again using canary build .176 then it should solve this issue.

I'd be happy to try it, I just need a tip about how to make it visible in my VS packages list, since even if I enable "Include pre-releases" NuGet only lists up to 85.24.6.176.

**Comment by Wagnerp** [comment]
@giduac 

This looks like a duplicate of issues: #1301, #1188 & #1145

**Comment by giduac** [comment]
1188 clearly about this

**Comment by Wagnerp** [comment]
Hi @giduac 

This is the PR I was talking about https://github.com/Krypton-Suite/Standard-Toolkit/pull/1288. I would hold off backporting it, as it does contain some significant code & designer changes, but it does solve this issue.

**Comment by giduac** [comment]
Hi @Smurf-IV,

Can you please have look at this to see if this is feasible to fix or that it would be better for @mbagattini to move forward to V90 Canary. 
Thanks

**Comment by Smurf-IV** [comment]
> > If @mbagattini can test again using canary build .176 then it should solve this issue.
> 
> I'd be happy to try it, I just need a tip about how to make it visible in my VS packages list, since even if I enable "Include pre-releases" NuGet only lists up to 85.24.6.176.

Just add `Canary` to your search string within the nuget package manager,
And then replace all `Krytpon` packages you are currently using with the latest:

![image](https://github.com/Krypton-Suite/Standard-Toolkit/assets/2418812/d49e4d36-7997-4d52-868f-a620f637aa08)

**Comment by Wagnerp** [comment]
> Hi @Smurf-IV,
> 
> Can you please have look at this to see if this is feasible to fix or that it would be better for @mbagattini to move forward to V90 Canary. Thanks

I would just add `canary` as pushing the changes into v85 now will break a lot of stuff for developers.

**Comment by giduac** [comment]
Hi @Wagnerp,
Please close as there's no further feedback, ty

**Comment by xpwang7** [comment]
marks
